db/migrate/20210103181832_connect_ad_with_city.rb

```diff
@@ -0,0 +1,13 @@
+class ConnectAdWithCity < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
```suggestion
class ConnectAdWithCity < ActiveRecord::Migration[6.1]
  Ad = Class.new(ActiveRecord::Base)
  City = Class.new(ActiveRecord::Base)
  
  def change
```

https://guides.rubyonrails.org/v3.2/migrations.html#using-models-in-your-migrations